### DISCUSSION TITLE
Support EconomyShopGUI within your api

### DISCUSSION THREAD

**Discussion by ImNotStable** [BG-Software-LLC/SuperiorSkyblock2#1556] [Pending Feature Requests] [answered]
Looking through the ShopGuiPlus API I believe you're connecting the prices within worth and I was curious if that was possible for you to within [EconomyShopGUI](https://dev.bukkit.org/projects/economyshopgui) aswell

**Comment by ImNotStable** [comment]
?

**Comment by OmerBenGera** [comment, marked as answer]
Added in latest dev build
